runtime/libpgmath/cmake/modules/LibmathUtils.cmake

```diff
@@ -1,17 +1,17 @@
-macro(libpgmath_say message_to_user)
-  message(STATUS "LIBPGMATH: ${message_to_user}")
+macro(libpgmath_say)
```

Comment:
Nice change there, but I'd rather see it mentioned in the commit message or moved to another commit.

Reply:
I have updated the commit message as you suggested. Thanks.